apps/reaper/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Reaper.MixProject do
   def project do
     [
       app: :reaper,
-      version: "2.0.42",
+      version: "2.0.43",
```

Comment:
Based on the changes I'm seeing. Looks like reaper may not need to be changed at all?